README.md

```diff
@@ -40,6 +40,6 @@ Aqua-park has two types of slides: for adult and for children.
 #### 5. `Slide` class
 Its `__init__` method takes 2 arguments:
    - `name` - string value, slide's name
-   - `limitation_class` - instance of `SlideLimitationValidator` class which sets restrictions on the use of the slide
+   - `limitation_class` - type of `SlideLimitationValidator` class which sets restrictions on the use of the slide
```

Comment:
Remove a change in the `README.md`, your pull request should contain only the `main.py` file. The task description relies on the suggestion that the variable name `limitation_class` is explicit enough to explain its content. Besides that, the most accurate description here would be the `subclass of SlideLimitationValidator class`.